indico/modules/events/sessions/views.py

```diff
@@ -25,7 +25,21 @@ class WPManageSessions(WPJinjaMixin, WPConferenceModifBase):
     sidemenu_option = 'sessions'
 
     def getJSFiles(self):
-        return (WPConferenceModifBase.getJSFiles(self) + self._asset_env['modules_sessions_js'].urls())
+        return WPConferenceModifBase.getJSFiles(self) + self._asset_env['modules_sessions_js'].urls()
 
     def getCSSFiles(self):
         return WPConferenceModifBase.getCSSFiles(self) + self._asset_env['sessions_sass'].urls()
+
+
+class WPDisplayMySessionsConference(WPJinjaMixin, WPConferenceDefaultDisplayBase):
+    template_prefix = 'events/sessions/'
+    menu_entry_name = 'my_sessions'
+
+    def _getBody(self, params):
+        return WPJinjaMixin._getPageContent(self, params)
+
+    def getJSFiles(self):
```

Comment:
Is this really needed? WPJinjaMixin doesn't override it so it should already use the correct one